.github/workflows/after-pullrequest.yml

```diff
@@ -16,12 +16,20 @@ jobs:
     name: Report Test Results
```

Comment:
We had a conversation in Meadowlark regarding this action, we should discuss this to see if we want to keep it and handle it this way.

This is failing at the moment since it's not downloading the artifact

Reply:
I remember that we had a conversation, but I don't remember the substance of that conversation. Uploading test results for future inspection seems like a useful thing.